decidim-admin/app/permissions/decidim/admin/permissions.rb

```diff
@@ -153,6 +153,8 @@ def user_action?
             !subject_user.admin? &&
             subject_user.roles.empty? &&
             Decidim::ImpersonationLog.active.where(admin: user).empty?
+        when :show_email
+          user.admin?
```

Comment:
Maybe it could be added to the `if user.admin? && admin_terms_accepted?`: 

```rb
allow! if show_participant_email_action?
```

And then in the `private` the method, so it would be easier to override:

```rb
private

def show_participant_email_action?
	user.admin?
end
```

Reply:
Thanks for the response, @agustibr ❤️ 
I was implementing your solution, but then I realized that I can just add a test to ensure that these permissions are correctly set in all scenarios 😬, so I just removed those lines.